functions/common/lib/tokens/ARBITRUM_NOVA_TOKENS.json

```diff
@@ -5,7 +5,7 @@
         "name": "Ethereum",
         "decimals": 18,
         "address": "0x0000000000000000000000000000000000000000",
-        "logoURI": "https://token.metaswap.codefi.network/assets/networkLogos/ethereum.svg"
```

Comment:
do not update compiled files manually, update it at https://github.com/HexlinkOfficial/Hexlink-Web/blob/main/functions/common/src/tokens/ARBITRUM_NOVA_TOKENS.json and rebuild the common pkg please